client/src/features/sessionsV2/components/SessionStatus/SessionStatus.tsx

```diff
@@ -0,0 +1,277 @@
+/*!
+ * Copyright 2024 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import cx from "classnames";
+import { Duration } from "luxon";
+import { ReactNode, useMemo } from "react";
+import {
+  CheckCircleFill,
+  Clock,
+  ExclamationCircleFill,
+  PauseCircleFill,
+  XCircleFill,
+} from "react-bootstrap-icons";
+import { Badge } from "reactstrap";
+import { Loader } from "../../../../components/Loader.tsx";
+import { TimeCaption } from "../../../../components/TimeCaption.tsx";
+import { NotebooksHelper } from "../../../../notebooks";
+import { SessionListRowStatusExtraDetails } from "../../../../notebooks/components/SessionListStatus.tsx";
+import { NotebookAnnotations } from "../../../../notebooks/components/session.types.ts";
+import { ensureDateTime } from "../../../../utils/helpers/DateTimeUtils.ts";
+import { MissingHibernationInfo } from "../../../session/components/status/SessionStatusText.tsx";
+import {
+  Session,
+  SessionStatusState,
+} from "../../../session/sessions.types.ts";
+import { SessionLauncher } from "../../sessionsV2.types.ts";
+
+export function SessionBadge({
+  children,
+  className,
+}: {
+  children: ReactNode;
+  className: string;
+}) {
+  return (
+    <Badge
+      className={cx(
+        "d-flex",
+        "align-items-center",
+        "gap-2",
+        "fs-small",
+        "fw-normal",
+        className
+      )}
+      pill={true}
+    >
+      {children}
+    </Badge>
+  );
+}
+interface ActiveSessionV2Props {
+  session: Session;
+}
+interface ActiveSessionTitleV2Props {
+  session: Session;
+  launcher?: SessionLauncher;
+}
+export function SessionStatusV2Label({ session }: ActiveSessionV2Props) {
+  const { annotations, status } = session;
+
+  const cleanAnnotations = useMemo(
+    () => NotebooksHelper.cleanAnnotations(annotations) as NotebookAnnotations,
+    [annotations]
+  );
+  const state = status.state;
+  const defaultImage = cleanAnnotations.default_image_used;
+
+  const badge =
+    state === "running" && defaultImage ? (
+      <SessionBadge className="border border-warning bg-warning-subtle">
+        <ExclamationCircleFill className="text-warning" size={16} />
```

Comment:
For all bootstrap icons, use class `"bi"` to align them with the surrounding text. Also, you can handle the margin here:

```suggestion
        <ExclamationCircleFill className={cx("bi", "me-1", "text-warning")}  size={16} />
```